examples/storage/couchdb/couchdb-petset-service.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: couchdb
+  name: kubetest001
+  namespace: default
+spec:
+  #clusterIP: 100.10.10.101
+  ports:
+  - name: in-tcp-port
```

Comment:
This was so the users could hit the service `kubetest001.defult.svc.cluster.local`.  Did this because I have not bundled up HAproxy in this example.